content/doc/administration/requirements/upgrade-java-guidelines.adoc

```diff
@@ -14,57 +14,57 @@ As with any upgrade, we recommend backing up `JENKINS_HOME` and testing the upgr
+NOTE: If you discover a previously unreported issue, please let us know: read <<./jenkins-on-java-11#discovering-issues-with-java-11,how to report a Java 11 Compatibility issue>> for guidance.
 
 One of the most important plugin upgrades is the plugin:workflow-support[Pipeline: Support plugin]: make sure that the version of the plugin is at least `3.0`. 
 
-NOTE: Stop all Pipeline jobs before upgrading this plugin, because this upgrade changes the serialization of Pipeline builds. As a general rule,even though Pipeline jobs are supposed to survive a Jenkins restart, it's always a better option to make sure that no Pipeline builds are in progress before any scheduled Jenkins maintenance.
+NOTE: Stop all Pipeline jobs before upgrading this plugin because this upgrade changes the serialization of Pipeline builds. As a general rule, even though Pipeline jobs are supposed to survive a Jenkins restart, it's always a better option to make sure that no Pipeline builds are in progress before any scheduled Jenkins maintenance.
 
 == Javax XML Bind libraries
 
 Some plugins use JAXB libraries provided by the JDK. However, the `java.xml.bind` and `javax.activation` modules are no longer included in OpenJDK 11, and plugins might fail if no replacement is offered.
 
-To fix this problem, we've bundled those libraries into a new detached plugin: plugin:jaxb[JAXB plugin]. When any Jenkins core more recent than `2.163` is running on Java 11, this plugin is automatically installed. However, if you manage your plugins outside Jenkins (for example, if you use `plugins.txt` in your Docker images), you might need to explicitly install the plugin.
+To fix this problem, we've bundled those libraries into a new detached plugin: plugin:jaxb[JAXB plugin]. When any Jenkins core more recent than `2.163` is running on Java 11, this plugin is automatically installed. However, if you manage your plugins outside Jenkins (for example, if you use `plugins.txt` in your Docker images), you might need to install the plugin explicitly.
 
 == JVM version on agents
 
 All agents must be running on the same JVM version as the controller (because of how controllers and agents communicate). If you're upgrading your Jenkins controller to run on Java 11, you also need to upgrade the JVM on your agents. 
 
-You can validate the version of each agent with the plugin:versioncolumn[Versions Node Monitors] plugin. This plugin provides information about the JVM version of each agent on the node management screen of your Jenkins instance. This plugin can also be configured to automatically disconnect any agent with an incorrect JVM version.
+You can validate the version of each agent with the plugin:versioncolumn[Versions Node Monitors] plugin. This plugin provides information about the JVM version of each agent on the node management screen of your Jenkins instance. You can also configure this plugin to disconnect any agent with an incorrect JVM version automatically.
```

Comment:
Placing the adverb nearer the verb helps me understand that the adverb applies to the verb.  Placing it after the prepositional phrase seems to make me work harder to understand that "automatically" applies to "disconnect"
```suggestion
You can validate the version of each agent with the plugin:versioncolumn[Versions Node Monitors] plugin. This plugin provides information about the JVM version of each agent on the node management screen of your Jenkins instance. You can also configure this plugin to automatically disconnect any agent with an incorrect JVM version.
```